contracts/protocore/Protocore.sol

```diff
@@ -114,4 +158,42 @@ contract Protocore is MosaicVersion, ValidatorSet, CoconsensusModule {
             openKernelHash
         );
     }
+
+
+    /* Internal Functions */
+
+    /**
+     * @notice It calculates vote message hash.
+     */
+    function hashVoteMessage(
+        bytes32 _transitionHash,
+        bytes32 _sourceBlockHash,
+        bytes32 _targetBlockHash,
+        uint256 _sourceBlockHeight,
```

Comment:
`_sourceBlockNumber` and `_targetBlockNumber`

Reply:
Done.